terraform/alerting/monitoring.tf

```diff
@@ -27,3 +27,49 @@ resource "google_monitoring_dashboard" "e2e" {
     null_resource.manual-step-to-enable-workspace
   ]
 }
+
+resource "google_monitoring_alert_policy" "five_xx" {
+  project      = var.project
+  display_name = "Elevated 5xx"
```

Comment:
This should be a string without space to make it easier to link to the playbook.

E.g. we can add "See [docs/${policy.display_name}](...)" to all alerts.